scss/atoms/modals/_modals.scss

```diff
@@ -0,0 +1,33 @@
+/*
+* ==========================================================================
+* Modals
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+.modal {
+	@include modal();
+
+	.modal-header {
+		@include modal-header();
+		h4 {
+			margin: 0;
+		}
+	}
+
+	.modal-body {
+		@include modal-body();
+	}
+
+	.modal-footer {
+		@include modal-footer();
+		@include clearfix();
+		& .btn {
```

Comment:
And that is very very weird?? 
```sass
.modal-footer {
  &  .btn  {
  }
}
```
whats wrong with
```sass
.modal-footer {
  .btn  {
  }
}
```
And either ways, I would rather have a separate class .modal-button .
And what @cvrebert  said, floating to right directly is very opinionated, that should be a variable as well

something like
```sass
.modal-button {
    float: map-get(modal-button,float)
}
```